test/foundry/Facets/UnlockFacet.t.sol

```diff
@@ -8,24 +8,39 @@ import { DepositFacet } from "src/facets/DepositFacet.sol";
 import { DiamondManagerFacet } from "src/facets/DiamondManagerFacet.sol";
 import { ERC20Mock } from "test/foundry/mocks/ERC20Mock.sol";
 import { StratosphereMock } from "test/foundry/mocks/StratosphereMock.sol";
+import { RewardsControllerMock } from "test/foundry/mocks/RewardsControllerMock.sol";
+import { LPercentages } from "src/libraries/LPercentages.sol";
 
 contract UnlockFacetTest is DiamondTest {
     LiquidMiningDiamond internal diamond;
     UnlockFacet internal unlockFacet;
     DepositFacet internal depositFacet;
     DiamondManagerFacet internal diamondManagerFacet;
+    RewardsControllerMock internal rewardsControllerMock;
     address depositFeeReceiver1 = makeAddr("depositFeeReceiver1");
     address depositFeeReceiver2 = makeAddr("depositFeeReceiver2");
     address unlockFeeReceiver1 = makeAddr("unlockFeeReceiver1");
     address unlockFeeReceiver2 = makeAddr("unlockFeeReceiver2");
 
+    // Stratosphere Mocks
+
+    address stratosphereMemberBasic = makeAddr("stratosphereMemberBasic");
+    address stratosphereMemberSilver = makeAddr("stratosphereMemberSilver");
+    address stratosphereMemberGold = makeAddr("stratosphereMemberGold");
+    address stratosphereMemberPlatinum = makeAddr("stratosphereMemberPlatinum");
+    address stratosphereMemberDiamond = makeAddr("stratosphereMemberDiamond");
+    address stratosphereMemberObsidian = makeAddr("stratosphereMemberObsidian");
+
+    uint256 public constant COOLDOWN_PERIOD = 72 * 3600; // 72 Hours
```

Comment:
better to write 72 hours here just for general rule. can you write this as 72 hours or check if it doesn't overflow using getter?

Reply:
Done